src/containers/Storage/utils/index.ts

```diff
@@ -40,3 +40,11 @@ export const getDegradedSeverity = (group: IStoragePoolGroup) => {
 
     return evaluate(group.Missing);
 };
+
+export const getUsageSeverity = generateEvaluator(80, 85);
+
+export const getUsage = (data: IStoragePoolGroup, step = 1) => {
```

Comment:
should check if step is not equals zero.

Reply:
I'm not convinced we should. With step=0 it just will result in NaN, which is correct. This argument is code-dependent, not data-dependent, so there is no chance of error because of wrong data, only because of a programmer’s error, and I don’t think checking will make it better somehow, since zero step doesn’t make sense anyway, and NaN is the correct result for it.